domains/ronan696.json

```diff
@@ -0,0 +1,14 @@
+{
+    "owner": {
+        "username": "ronan696",
+        "email": "ronan696@gmail.com"
+    },
+    "record": {
+        "CNAME": "ronan696.github.io",
+        "MX": [
+            "mx1.improvmx.com",
+            "mx2.improvmx.com"
+          ],
+        "TXT": "v=spf1 include:spf.improvmx.com ~all"
```

Comment:
You can't use CNAME with other records

Reply:
Apologies. Got rid of the other records. Thanks!